lib/main.js

```diff
@@ -327,6 +411,15 @@ function signText(text) {
       return ERROR_INTERNAL;
     }
 
+    // Get certificate slot
+    let slot = ctypes.cast(cert, CERTCertificateStr.ptr).contents.slot;
+    // Check provided password if cert slot needs login
+    if(PK11_CheckUserPassword(slot, certName.password) != SECSuccess) {
+      log("Token password verification failed");
```

Comment:
I will change it to reference function name to be consistent with other log messages.